### DISCUSSION TITLE
How do I modify a password's permissions to be read-only?

### DISCUSSION THREAD

**Discussion by thomasmost** [planetscale/discussion#229] [FAQs] [answered]
I have created a dedicated password for our Metabase instance. All passwords, I see, "Can Read, Write & Administer".... how can I modify the permissions of a password (mysql user) to be read-only?

**Comment by jgreet** [comment, marked as answer]
It isn't currently possible to modify a password's permissions, or create a password with permissions other than Admin. 

More permission options are coming soon.

**Reply by jgreet** [reply]
Read only passwords shipped today:
https://docs.planetscale.com/concepts/password-roles

**Comment by thomasmost** [comment]
Holy mackerel! Terrific!

Thank you so much, John :)

Any word on who we could talk to at Planetscale about your sustainability
data? (Separate issue, I know, but I opened a ticket inquiring about this
and didn't hear back)

**Reply by jgreet** [reply]
Weird, did you log in to our support portal and check there? I'm pretty sure I answered that one.

https://support.planetscale.com/hc/en-us/signin

**Comment by thomasmost** [comment]
Oh, I see yes!

Very odd, I don't seem to have ever received an email for/with that message.

Would you mind sharing Dan Burrill's email address with me?

**Reply by jgreet** [reply]
👍  Sent you another message via Zendesk.
